dataframe-jupyter/src/main/kotlin/org/jetbrains/kotlinx/dataframe/jupyter/Integration.kt

```diff
@@ -282,13 +293,16 @@ internal class Integration(private val notebook: Notebook, private val options:
         import("org.jetbrains.kotlinx.dataframe.impl.codeGen.urlCodeGenReader")
 
         addTypeConverter(object : FieldHandler {
+
             override val execution = FieldHandlerFactory.createUpdateExecution<Any> { instance, property ->
+                // TODO check property type first, then instance, Issue #1245
                 when (instance) {
                     is AnyCol -> updateAnyColVariable(instance, property, codeGen)
                     is ColumnGroup<*> -> updateColumnGroupVariable(instance, property, codeGen)
                     is AnyRow -> updateAnyRowVariable(instance, property, codeGen)
                     is AnyFrame -> updateAnyFrameVariable(instance, property, codeGen)
                     is ImportDataSchema -> updateImportDataSchemaVariable(instance, property)
+                    is GroupBy<*, *> -> updateGroupByVariable(instance, property, codeGen)
```

Comment:
I have one question here: do we need the same support for Pivot as for GroupBy? At least create a ticket and make a separate fix? or validate with test